modules/raft/src/main/java/org/apache/ignite/internal/raft/util/SharedLogStorageFactoryUtils.java

```diff
@@ -33,10 +34,15 @@ public class SharedLogStorageFactoryUtils {
     public static final String LOGIT_STORAGE_ENABLED_PROPERTY = "LOGIT_STORAGE_ENABLED";
 
     /** Creates a LogStorageFactory with the {@link DefaultLogStorageFactory} implementation. */
+    @TestOnly
     public static LogStorageFactory create(String nodeName, Path lazyLogStoragePath) {
```

Comment:
Could you please also rename that `lazyLogStoragePath` to `logStoragePath`? It seems that 'lazy' is not relevant anymore

Reply:
Sure